deltachat-ios.xcodeproj/project.pbxproj

```diff
@@ -2513,7 +2527,7 @@
 				INFOPLIST_FILE = DcWebxdcWidget/Info.plist;
 				INFOPLIST_KEY_CFBundleDisplayName = DcWebxdcWidget;
 				INFOPLIST_KEY_NSHumanReadableCopyright = "Copyright © 2024 merlinux GmbH. All rights reserved.";
-				IPHONEOS_DEPLOYMENT_TARGET = 18.1;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.6;
```

Comment:
great - so widgets will work on all phones but the ones only having ios12 (which is about being deprecated anyways soon)